asv/benchmarks/setup/bench_model.py

```
@@ -57,20 +120,64 @@ class FastInitializeModel:
+
+            for _ in range(self.repeat):
+                model_builder_begin = time.perf_counter()
+                builder = Example.create_model_builder(robot, num_envs, randomize=True, seed=123)
+
+                # finalize model
+                model = builder.finalize()
+                wp.synchronize_device()
+                model_builder_end = time.perf_counter()
+                timings_model_builder.append(model_builder_end - model_builder_begin)
+
+                solver_begin = time.perf_counter()
```

Ruff (0.12.2)

Local variable `solver_begin` is assigned to but never used

Remove assignment to unused variable `solver_begin`

(F841)